### PR TITLE
handle org delete account

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
related PR : https://github.com/yalla-coop/cost-of-living-support/pull/106/
related issue : https://github.com/yalla-coop/cost-of-living-support/issues/37
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/modules/user/controllers/delete-user.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+      userRoles.SUPER_ADMIN === role ||
+      (userRoles.ADMIN === role && adminId === id)
+    ) {
+      await User.deleteUser({ id });
+    } else {
+      throw Boom.forbidden();
+    }
     res.send();
   } catch (error) {
     next(error);
```
Comment (RamyAlshurafa) [inline server/modules/user/controllers/delete-user.js:21]: this doesn't match the same file in CoL

File: server/modules/organisation/use-cases/delete-organisation.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import * as Organisation from '../model';
+
+function deleteOrganisation({ id }) {
+  return Organisation.updateOrganisationToDeleted(id);
+}
+
+export default deleteOrganisation;
```
Comment (RamyAlshurafa) [inline server/modules/organisation/use-cases/delete-organisation.js:7]: doesn't match the same file in CoL
Make sure to go through all the backend files again please

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
- [ ] add migration files
- [x] get the changes from this commit https://github.com/yalla-coop/cost-of-living-support/pull/106/commits/8279a28c85b51494663fbcd1a5d5979ebd3a4ad7
- [x] I deleted my account but the organization was not deleted https://www.loom.com/share/a1013e37b2ca4bb8a9ff1528006dbdec

**Comment by fadeomar** [comment]
hey @RamyAlshurafa can you work on the migration file point, please?

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**